rotkehlchen/globaldb/handler.py

```diff
@@ -1978,3 +1979,31 @@ def get_assets_in_same_collection(identifier: str) -> tuple[Asset, ...]:
             return (Asset(identifier),)
 
         return collection_assets
+
+    @staticmethod
+    def get_asset_from_exchange_name(exchange: Location, symbol: str, default: str) -> str:
+        """Returns the asset's identifier from the ticker symbol of the given exchange according to
```

Comment:
correct me if I'm wrong. This function is only used in `converters.py` and there you always do `symbol_to_asset_or_token(output_of_this_function)`. Call `symbol_to_asset_or_token` here

Reply:
I did that earlier, but there is `asset_from_woo` where only the identifier is needed in the first call. That's also why I forgot to remove the `May Raise: UnknownAsset` from their docstrings.